dm/dm.go

```diff
@@ -166,9 +166,19 @@ func execute() {
 	switch args[0] {
 	case "templates":
 		path := fmt.Sprintf("registries/%s/types", *templateRegistry)
+		if *regexString != "" {
```

Comment:
I'm not sure how much content they have yet to pull from the client, so until we know they're done pulling content from it, I think it makes sense to make it work correctly. It was an easy change, so made sense to do it, especially to make sure the server handles the regex in the query string correctly.